[2024-12-26T02:03:33.770+0200] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T02:03:33.801+0200] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simulator_5_dag.send_to_api_5 scheduled__2024-12-25T23:00:00+00:00 [queued]>
[2024-12-26T02:03:33.819+0200] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simulator_5_dag.send_to_api_5 scheduled__2024-12-25T23:00:00+00:00 [queued]>
[2024-12-26T02:03:33.823+0200] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-26T02:03:33.891+0200] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): send_to_api_5> on 2024-12-25 23:00:00+00:00
[2024-12-26T02:03:33.900+0200] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'simulator_5_dag', 'send_to_api_5', 'scheduled__2024-12-25T23:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/Dynamic_Dags.py', '--cfg-path', '/tmp/tmpx4rwxtuu']
[2024-12-26T02:03:33.898+0200] {standard_task_runner.py:72} INFO - Started process 188537 to run task
[2024-12-26T02:03:33.903+0200] {standard_task_runner.py:105} INFO - Job 19: Subtask send_to_api_5
[2024-12-26T02:03:34.033+0200] {task_command.py:467} INFO - Running <TaskInstance: simulator_5_dag.send_to_api_5 scheduled__2024-12-25T23:00:00+00:00 [running]> on host ali-nasser.gizasystems.local
[2024-12-26T02:03:34.255+0200] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='simulator_5_dag' AIRFLOW_CTX_TASK_ID='send_to_api_5' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-25T23:00:00+00:00'
[2024-12-26T02:03:34.257+0200] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-26T02:03:34.259+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-26T02:03:34.263+0200] {logging_mixin.py:190} INFO - Current task name:send_to_api_5 state:running start_date:2024-12-26 00:03:33.804064+00:00
[2024-12-26T02:03:34.265+0200] {logging_mixin.py:190} INFO - Dag name:simulator_5_dag and current dag run status:running
[2024-12-26T02:03:34.268+0200] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T02:03:34.290+0200] {logging_mixin.py:190} WARNING - /mnt/e/Assignments/Airflow/airflow/airflow_env/lib/python3.12/site-packages/django/db/models/fields/__init__.py:1665 RuntimeWarning: DateTimeField SimulatorResult.timestamp received a naive datetime (2024-12-26 00:03:34) while time zone support is active.
[2024-12-26T02:03:34.311+0200] {logging_mixin.py:190} INFO - Response saved for Simulator 5: {'asset_id': 1, 'attribute_id': 'output_1', 'timestamp': '2024-12-26T00:03:34Z[UTC]', 'value': 395.8634738360771}
[2024-12-26T02:03:34.314+0200] {python.py:240} INFO - Done. Returned value was: None
[2024-12-26T02:03:34.330+0200] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T02:03:34.332+0200] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=simulator_5_dag, task_id=send_to_api_5, run_id=scheduled__2024-12-25T23:00:00+00:00, execution_date=20241225T230000, start_date=20241226T000333, end_date=20241226T000334
[2024-12-26T02:03:34.378+0200] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-26T02:03:34.380+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-26T02:03:34.383+0200] {logging_mixin.py:190} INFO - Dag name:simulator_5_dag queued_at:2024-12-26 00:00:01.566802+00:00
[2024-12-26T02:03:34.388+0200] {logging_mixin.py:190} INFO - Task hostname:ali-nasser.gizasystems.local operator:PythonOperator
[2024-12-26T02:03:34.398+0200] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T02:03:34.433+0200] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T02:03:34.439+0200] {local_task_job_runner.py:245} INFO - ::endgroup::
